intro-HPC/ch_FAQ.tex

```diff
@@ -56,3 +56,17 @@ \section{Can I use multple different SSH key pairs to connect ot my VSC account?
 
 Yes, and this is recommendend when working from different computers. Please see
 \autoref{sec:adding-multiple-keys} on how to do this.
+
+\section{I want to use software that isn't installed on the clusters}
+
+\ifgent
+Please fill out the details about the software and why you need it in this form:
+\url{https://www.ugent.be/hpc/en/support/software-installation-request}.
```

Comment:
When submitting the form, a mail will be sent to `\hpcinfo` containing all the provided information. The HPC team will look into your request as soon as possible you and contact you when the installation is done or if further information is required.